### PR TITLE
Repositories can now be created

### PR DESCRIPTION
**Pull request by maiyuki**

### CODE REVIEW

File: phabfive/diffusion.py [hunk 1]
```diff
@@ -10,12 +10,33 @@
...
 class Diffusion(Phabfive):
     def __init__(self):
         super(Diffusion, self).__init__()
 
+    def create_repository(self, name=None):
+        """Phabfive wrapper that connects to Phabricator and creates repositories.
+
+        :type name: str
+
+        :rtype: str
```
Comment (holmboe) [inline phabfive/diffusion.py]: Currently returns nothing/`None`

**Comment by holmboe** [comment]
Also update README with a checked box for adding a repository.

File: phabfive/cli.py [hunk 2]
```diff
@@ -130,15 +131,19 @@ def run(cli_args, sub_args):
...
+                if sub_args["list"]:
+                    if sub_args["active"]:
+                        status = ["active"]
+                    elif sub_args["inactive"]:
+                        status = ["inactive"]
+                    else:
+                        status = REPO_STATUS_CHOICES
+
+                    d.print_repositories(status=status, url=sub_args["--url"])
+
```
Comment (Grokzen) [inline phabfive/cli.py]: Lint: There should not be blank lines before a `elif or else` even in deeply nested logic like this.

File: phabfive/diffusion.py (hunk 1 above)
Comment (maiyuki) [reply phabfive/diffusion.py]: Fixed

File: phabfive/cli.py (hunk 2 above)
Comment (maiyuki) [reply phabfive/cli.py]: Fixed

**Pull request merged**